src/app/core/rest/video.service.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {HttpClient} from '@angular/common/http';
+import {Injectable} from '@angular/core';
+import {Observable} from 'rxjs';
+import {VideoMetaData} from '../store/videos/video.model';
+
+@Injectable()
+export class VideoService {
+  constructor(private httpClient: HttpClient) {}
+
+  public getVideoMetadata(id: string, apiKey: string): Observable<VideoMetaData> {
+    return this.httpClient.get<VideoMetaData>(VideoService.getVideoApi(id, apiKey));
+  }
+
+  private static getVideoApi(id: string, apiKey: string): string {
+    return `https://www.googleapis.com/youtube/v3/videos?part=snippet&id=${id}&key=${apiKey}`;
```

Comment:
We should distinguish between services communicating with our REST API and the ones communicating with external APIs. We started putting the former ones in `core/rest` modules and the latter in `core/api`. I know, the naming is not very clear but let's just keep them separately.

So what I suggest is moving this file to `core/api/video` folder.